### PR TITLE
make the reports folder

### LINKED ISSUE #7: Report folder setup
Please setup the report's folder with the proposal.md. Many thanks!

### PR DESCRIPTION
**Pull request by MoNorouzi23**
Please review and merge it!

### CODE REVIEW

**Review by 786213750 — APPROVED** [review]
Looks great!

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+## Code of Conduct
+#### DSCI 532 - Group 04
+##### March 25, 2024
+
+#### Our Standards
+
+Every member of our community has the right to have their identity respected. DSCI 522 Group 17 is dedicated to providing a positive experience for everyone, regardless of age, gender identity and expression, sexual orientation, disability, physical appearance, body size, ethnicity, nationality, race, or religion (or lack thereof), education, or socio-economic status.
```
Comment (s-voon) [inline CODE_OF_CONDUCT.md:7]: DSCI 532 Group 4 instead of DSCI 522 Group 17

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+#### Expected Behavior
+
+All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person. In order to foster a positive and professional learning environment we encourage the following kinds of behaviours in all Group 17 events and platforms:
```
Comment (s-voon) [inline CODE_OF_CONDUCT.md:12]: Group 4

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (s-voon) [inline CODE_OF_CONDUCT.md]:
```suggestion
Every member of our community has the right to have their identity respected. DSCI 532 Group 4 is dedicated to providing a positive experience for everyone, regardless of age, gender identity and expression, sexual orientation, disability, physical appearance, body size, ethnicity, nationality, race, or religion (or lack thereof), education, or socio-economic status.
```

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (s-voon) [inline CODE_OF_CONDUCT.md]:
```suggestion
All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person. In order to foster a positive and professional learning environment we encourage the following kinds of behaviours in all Group 4 events and platforms:
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- Sustained disruption of online community discussions, in-person presentations, or other in-person events
+- Continued one-on-one communication after requests to cease
+- Other conduct that is inappropriate for a professional audience including people of many different backgrounds
+
+Community members asked to stop any inappropriate behavior are expected to comply immediately.
+
+
+#### Consequences
+
+Participants who are asked to stop any inappropriate behavior are expected to comply immediately. This applies to any Group 17 meetings and platforms, either online or in-person. If a participant engages in behavior that violates this code of conduct, the team may warn the offender and or engage the DSCI 522 instructor to investigate the Code of Conduct violation and impose appropriate sanctions.
```
Comment (s-voon) [inline CODE_OF_CONDUCT.md]:
```suggestion
Participants who are asked to stop any inappropriate behavior are expected to comply immediately. This applies to any Group 4 meetings and platforms, either online or in-person. If a participant engages in behavior that violates this code of conduct, the team may warn the offender and or engage the DSCI 532 instructor to investigate the Code of Conduct violation and impose appropriate sanctions.
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+
+#### Incident Reporting Guidelines
+
+If someone violates the Code of Conduct during a Group 17 event or in an online space outside of organised events, we ask that you ask the person to stop. If they do not stop the inappropriate behavior immediately, raise the issue with the team and the behavior will be addressed by the rest of the team members. If you do not feel the behavior has been adequately addressed, or it has still not stopped, report it to the Instructor(s). The current instructor(s) of DSCI 522 is(are) Tiffany Timbers. An email message to the email address listed in the course syllabus will be seen by the instructor. To report a DSCI 522 Group 17 code of conduct violation that will not be seen by your instructor, an alternative contact person is Milad Maymay, Director, Program Operations and Student Management, MDS Vancouver. His email is maymay@science.ubc.ca.
```
Comment (s-voon) [inline CODE_OF_CONDUCT.md]:
```suggestion
If someone violates the Code of Conduct during a Group 4 event or in an online space outside of organised events, we ask that you ask the person to stop. If they do not stop the inappropriate behavior immediately, raise the issue with the team and the behavior will be addressed by the rest of the team members. If you do not feel the behavior has been adequately addressed, or it has still not stopped, report it to the Instructor(s). The current instructor(s) of DSCI 532 is(are) Joel Ostblom. An email message to the email address listed in the course syllabus will be seen by the instructor. To report a DSCI 532 Group 4 code of conduct violation that will not be seen by your instructor, an alternative contact person is Milad Maymay, Director, Program Operations and Student Management, MDS Vancouver. His email is maymay@science.ubc.ca.
```

**Review by s-voon — CHANGES_REQUESTED** [review]
Make a few changes, specifically on the code of conduct.md file. Please review it! Thanks!

**Comment by WaleedMahmood1** [comment]
Thanks for the review @s-voon! I have accepted and committed your review into the `CODE_OF_CONDUCT.md` file.

**Review by WaleedMahmood1 — APPROVED** [review]
Project structure looks complete now, I believe. We can merge this pull request with main now.

**Review by WaleedMahmood1 — APPROVED** [review]
Thanks for the review, I have committed these changes!

**Review by WaleedMahmood1 — APPROVED** [review]

**Review by s-voon — APPROVED** [review]

**Pull request merged**